package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cuttle",
-  "version": "4.0.6",
+  "version": "4.0.7",
```

Comment:
Can you please do the same in `package.json`? `package-lock.json` is auto generated/updated when you install the dependencies based on `package.json` itself. That's the main file, so the version there is what's important